manticore/platforms/linux.py

```diff
@@ -447,20 +450,42 @@ def _init_cpu(self, arch):
         self._function_abi = CpuFactory.get_function_abi(cpu, 'linux', arch)
         self._syscall_abi = CpuFactory.get_syscall_abi(cpu, 'linux', arch)
 
-    def _execve(self, program, argv, envp):
+    def _find_symbol(self, name):
+        symbol_tables = [s for s in self.elf.iter_sections()
```

Comment:
We can freely go with creating a generator here instead of a list (change `[...]` to `(...)`) as we do `return None` if we iterate over nothing anyway.

Reply:
ok